src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -50,8 +50,8 @@ object Boot extends App {
       dataSource.shutdown()
     }
 
-    val methodRepoConfig = conf.getConfig("methodrepo")
-    val service = system.actorOf(RawlsApiServiceActor.props(swaggerService, WorkspaceService.constructor(dataSource, new GraphWorkspaceDAO(), new GraphEntityDAO(), new GraphMethodConfigurationDAO(), methodRepoConfig.getString("server"))), "rawls-service")
+    val xSvcDAO = new RealExternalServiceDAO(conf.getConfig("methodrepo"),conf.getConfig("execsvc"))
```

Comment:
Let's stay away from non-standard abbreviations please. This should be executionServiceDAO.